x-pack/plugins/fleet/server/services/artifacts/artifacts.ts

```diff
@@ -84,55 +86,95 @@ export const createArtifact = async (
   return esSearchHitToArtifact({ _id: id, _source: newArtifactData });
 };
 
-export const bulkCreateArtifacts = async (
-  esClient: ElasticsearchClient,
+// Max length in bytes for artifacts batch
+const MAX_LENGTH_BYTES = 2_000;
```

Comment:
It could be a config value similar to these: https://github.com/elastic/kibana/blob/717358cef6029779167cf0d789fec775617d15b7/x-pack/plugins/fleet/common/types/index.ts#L17

Reply:
2k seems low. Maybe ask the ES team?

Reply:
Did we get any suggestion from the ES team on batch size?
If not, is there a way to test with a cluster that has many artifacts to see what are the typical artifact sizes?

Reply:
I'm waiting for an answer, as soon as I get it, will update the pr